sdk/ml/azure-ai-ml/tests/dsl/unittests/test_dsl_pipeline_with_specific_nodes.py

```diff
@@ -186,6 +186,45 @@ def train_with_sweep_in_pipeline():
         omit_fields = ["trial"]
         assert pydash.omit(sweep_node_dict, *omit_fields) == pydash.omit(sweep_node_dict_from_rest, *omit_fields)
 
+    def test_dsl_pipeline_with_sweep_distributed_component_setting_instance_type(self) -> None:
+        yaml_file = "./tests/test_configs/components/helloworld_component_pytorch.yml"
+
+        @dsl.pipeline(force_rerun=True)
+        def train_with_sweep_in_pipeline(raw_data):
+            component_to_sweep: CommandComponent = load_component(source=yaml_file)
+            cmd_node: Command = component_to_sweep(
+                component_in_number=Choice([2, 3, 4, 5]), component_in_path=raw_data
+            )
+            sweep_job: Sweep = cmd_node.sweep(
+                primary_metric="AUC",  # primary_metric,
+                goal="maximize",
+                sampling_algorithm="random",
+            )
+            sweep_job.resources.instance_type = "cpularge"
```

Comment:
Add comment that this is not the ideal way.

Reply:
added